model/post_list.go

```diff
@@ -18,6 +18,8 @@ type PostList struct {
 	HasNext bool `json:"has_next"`
 	// HasInaccessiblePosts tells if there are inaccessible posts, past the cloud limit.
 	HasInaccessiblePosts bool `json:"has_inaccessible_posts"`
+	// If there are inaccessible posts, FirstInaccessiblePostTime is the time of the latest inaccessible post
+	FirstInaccessiblePostTime int64 `json:"first_inaccessible_post_time"`
```

Comment:
Yes, that works for me. I think we can assume no code already depends on HasInaccessiblePosts since this only affects cloud instances and I don't think any staff has written any code depending on HasInaccessiblePosts yet.